src/components/form/index.js

```diff
@@ -1,45 +1,69 @@
 import React from 'react';
 import './index.css';
 
+/**
+ * .Form Component to create an input.
+ */
 class Form extends React.Component {
```

Comment:
Let's make the `Form` component more generic and may be create components for searching and adding todo. They will contain specific logic regarding searching and creating todo and will be based on the generic Form component.